modules/network-observability-create-network-policy.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+
+// * networking/network_observability/configuring-operators.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-network-policy_{context}"]
+= Secure Network Observability with a network policy using the web console
+You may need to create a network policy to secure ingress traffic to the 'netobserv' namespace. Using the web console, you can create a network policy 
+
+.Procedure
+. Navigate to *Networking* -> *NetworkPolicies*.
+. Select the `netobserv` project from the *Project* dropdown menu.
+. Name the policy. For this example, the policy name is `allow-ingress`.
+. Click the *Add ingress rule* button twice to create two ingress rules, and make the following specifications in the form:
+.. Make the following specifications for the first *Ingress rule*:
+... From the *Add allowed source* dropdown menu, select *Allow pods from inside the cluster*. 
+... Click *+ Add namespace selector*. 
+... Add the label, `kubernetes.io/metadata.name`, and the selector, `openshift-console`. 
+.. Make the following specifications for the second *Ingress rule*:
+... From the *Add allowed source* dropdown menu, select *Allow pods from inside the cluster*.
+... Click *+ Add namespace selector*. 
+... Add the label, `kubernetes.io/metadata.name`, and the selector, `openshift-monitoring`.
+
+.Optional
+You can edit the network policy using the YAML editor in the web console. The YAML file should be like the following sample:
+
+[id="network-observability-network-policy-sample_{context}"]
+.Sample network policy
+[source, yaml]
+----
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: lockdown-netobserv
+  namespace: netobserv
+spec:
+  podSelector: {}
+  ingress:
+    - from:
+        - podSelector: {}   <1>
+        - podSelector: {}
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: openshift-console
+        - podSelector: {}
+          namespaceSelector:
+            matchLabels:
+              kubernetes.io/metadata.name: openshift-monitoring
+  policyTypes:
+    - Ingress
+status: {}
+----
+<1> If the `namespaceSelector` is not specified, the `podSelector` defaults to the specified namespace. In this sample, the namespace is `netobserv`.
+
+.Verification
+. Navigate to *Observe* -> *Network Traffic*. 
+. View the *Traffic Flows* tab, or any tab, to verify data is displayed.
+. Navigate to *Observe* -> *Dashboards* the NetObserv/Health selection and verify that the flows are being ingested and sent to Loki, which is represented in the first graph.
```

Comment:
```suggestion
. Navigate to *Observe* -> *Dashboards* the NetObserv/Health selection and verify that the flows are being received and sent to Loki. This activity is represented in the first graph.
```
Haven't seen "ingested" so curious if it can just be received. If there's other processing happening, then I can see using "ingested." Also not sure if the activity is shown on the graph, or if Loki is? For your consideration.

Reply:
Yeah, its not just receiving. "Ingest" is a common term used in Network Observability and Logging. First the flows are ingested and enriched with kubernetes data, then sent to Loki or Kafka for further processing.